acs5_IL_county_calculatedpovertyrate.csv

```diff
@@ -0,0 +1,103 @@
+,NAME,Total Population,Poverty Population,state,county,Poverty Rate
```

Comment:
Is this the final result of the script? Or is this the raw data that was pulled down from the census?

Reply:
Removed file, replaced with json output